robottelo/ui/browser.py

```diff
@@ -42,12 +42,12 @@ def browser():
                 else webdriver.Chrome(
                     executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'ie':
+        elif webdriver_name == 'internet explorer':
             return (
                 webdriver.Ie() if settings.webdriver_binary is None
                 else webdriver.Ie(executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'edge':
+        elif webdriver_name == 'MicrosoftEdge':
```

Comment:
And the variable is called 'webdriver_name', not 'browserName', that's my concern.